app/data/mwss_4.json

```diff
@@ -0,0 +1,3729 @@
+                                    "type": "General"
+                                }
+                            ],
+                            "title": "Fortnightly Pay"
+                        }
+                    ],
+                    "type": "questionnaire"
+                },
+                {
+                    "id": "fortnightly-pay-block-23",
+                    "sections": [
+                        {
+                            "description": "<p>1 December 2016 to 31 December 2016</p>",
+                            "id": "fortnightly-pay-section-23",
+                            "number": "3",
+                            "questions": [
+                                {
+                                    "answers": [
+                                        {
+                                            "description": "",
+                                            "guidance": "",
+                                            "id": "fortnightly-pay-answer-23-0",
+                                            "label": "",
+                                            "mandatory": true,
+                                            "options": [
+                                                {
+                                                    "label": "More temporary staff",
+                                                    "value": "More temporary staff"
+                                                },
+                                                {
+                                                    "label": "Fewer temporary staff",
+                                                    "value": "Fewer temporary staff"
+                                                },
+                                                {
+                                                    "label": "No significant change",
+                                                    "value": "No significant change"
+                                                }
+                                            ],
+                                            "type": "Radio"
+                                        }
+                                    ],
+                                    "description": "",
+                                    "id": "fortnightly-pay-question-23",
+                                    "number": "3.6",
+                                    "title": "Did any significant changes occur to the <em>number of temporary fortnightly paid employees</em>?",
+                                    "type": "General"
+                                }
+                            ],
+                            "title": "Fortnightly Pay"
+                        }
+                    ],
+                    "type": "questionnaire"
+                },
+                {
+                    "id": "fortnightly-pay-bloc
```

Comment:
I didn't answer the Pay Pattern questions, and when I got to this slide and tried to POST the screen I got a 404 error, probably because it can't work out the next location. The weekly, fortnightly, monthly etc sections should only be displayed if they chose the matching option for the Pay Pattern question.